plugins/repository-hdfs/build.gradle

```diff
@@ -48,7 +48,7 @@ dependencies {
   compile "org.apache.hadoop:hadoop-hdfs:${versions.hadoop2}"
   compile "org.apache.hadoop:hadoop-hdfs-client:${versions.hadoop2}"
   compile 'org.apache.htrace:htrace-core4:4.0.1-incubating'
-  compile 'com.google.guava:guava:11.0.2'
+  compile "com.google.guava:guava:${versions.guava}"
```

Comment:
We can not make this change. This assumes full backwards compatibility with the version that Hadoop depends on and this is not the case. Trying to upgrade the Guava version used by Hadoop is quite the [saga](https://issues.apache.org/jira/browse/HADOOP-10101) with a very long history, exactly because there are compatibility problems.